test_scripts/API/VehicleData/SubscribeVehicleData/003_RPC_parameter_already_subscribed_Result_IGNORED_flow.lua

```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------------------------------
+-- Item: Use Case: request is allowed by policies but app is already subscribed for specified parameter
+--
```

Comment:
@AByzhynar, add link to the requirement when they will be available.

Reply:
@AStasiuk Fixed in 3b703abdab25b3e13b150954e8a8c707f40a3b45